airbyte-integrations/connectors/source-cart/source_cart/streams.py

```diff
@@ -33,8 +33,9 @@
 class CartStream(HttpStream, ABC):
     primary_key = "id"
 
-    def __init__(self, start_date: str, store_name: str, **kwargs):
+    def __init__(self, start_date: str, store_name: str, end_date: str = None, **kwargs):
         self._start_date = start_date
+        self._end_date = end_date
```

Comment:
self.end_date = end_date if end_date else now()

Reply:
this value will be added to a request query if it is not empty. And it will be able to complicate a server handling (Perhaps this query is converted to some SQL request and an addl conditions can raise a response time from DB). Thus it would be better if all query default parameters are skipped from our side.